### ISSUE TITLE
[BUG] ImportError: cannot import name 'pyShelly' from 'pyShelly'

### ISSUE THREAD

**Issue by mleiber** [StyraHem/ShellyForHASS#693]
## Environment

- ShellyForHASS release with the issue: 1.0.0
- Last working ShellyForHASS release (if known): n/a
- Home Assistant Core release with the issue: 2022.7.3
- Operating environment (Home Assistant/Supervised/Docker/venv): HA OS 8.2

## Describe the bug

This appears in the log and none of the Shelly devices is created. Looks like it is related to the most recent update of HA to 2022.7.3. I am pretty sure I have not seen this with 2022.7.2 and earlier. I opened the terminal to HA and it seems as though pyShelly is not installed at all. Also, the location  for python3.10 is /usr/lib/python3.10, not /usr/local/lib/python3.10. May be related to the issue. 

```
Traceback (most recent call last):
  File "/config/custom_components/shelly/__init__.py", line 298, in start_up
    from pyShelly import pyShelly
ImportError: cannot import name 'pyShelly' from 'pyShelly' (/usr/local/lib/python3.10/site-packages/pyShelly/__init__.py)
```

## Traceback/Error logs

```txt

```

## Additional context

Downgraded to 2022.7.2 and the issue is gone.

**Comment by bcutter** [comment]
Why did you close this issue @mleiber ? A HA downgrade is not a solution, right 😄 
Is it maybe solved meanwhile by another HA update (>= 2022.7.4)?

**Comment by mleiber** [comment]
sorry, completely forgot about this. I think the issue was a somehow incomplete update to 22.7.3. I believe at the time it was then solved with the next release. Haven't seen anyhing like this later on.

Again, mea culpa to not have mentioned the remedy when closing the issue.
